### ISSUE TITLE
filter news articles by language in reference fields

### ISSUE THREAD

**Issue by SvSven** [equinor/energyvision#446]
The Sanity localization plugin adds a `_lang` field to translated documents, but not on the source document.
We need to figure out a way to filter the reference field by language in order to not have a endless list of pages to scroll through when making an internal link.

Code below used to filter topic pages, this does not work for English news articles due to missing `_lang` field.

```
          options: {
            filter: ({ document: { title: title = '', _lang } }: any): SchemaType.ReferenceFilter => ({
              filter: `title != $title && (_type == $routeLang || (_type == 'news' && _lang == $lang))`,
              params: { title: title, routeLang: `route_${_lang}`, lang: _lang },
            }),
          },
```

**Comment by SvSven** [comment]
Seems like the missing `_lang` field has been fixed in the intl plugin, so more recent news articles have the field but older ones do not, and the reference field will fail with a broken filter error.

**Comment by nilsml** [comment]
Fixed by #725
